app/controllers/peer_reviews_controller.rb

```diff
@@ -157,6 +158,43 @@ def csv_upload_handler
     redirect_to action: 'index', assignment_id: assignment_id
   end
 
+  # ===========================================================================
+  # Forwarders
+  # ===========================================================================
+
+  def test
+    assignment = Assignment.find(params[:assignment_id])
+    parent_assignment = assignment.parent_assignment
+    pr = PeerReview.find(params[:id])
+    path = edit_assignment_submission_result_path(parent_assignment, pr.reviewee, pr.result)
+
+    #redirect_to path
+
+    #render 'results/edit'
+
+    # render template: 'results/edit', :locals => {
+    #     result: pr.result,
+    #     assignment: parent_assignment
+    # }
+
+    #render :template => 'results/view_marks', :locals => { assignment: assignment, result: pr.result }
+
+    # controller_you_want = ResultsController.new
+    # controller_you_want.request = request
+    #
+    # debugger
+    #
+    # controller_you_want.request.params["controller"] = "results"
+    # controller_you_want.request.params["action"] = "edit"
+    # controller_you_want.request.params["assignment_id"] = parent_assignment.id.to_s
+    # controller_you_want.request.params["submission_id"] = params[:submission_id].to_s
+    # controller_you_want.request.params["id"] = pr.result.id.to_s
+    #
+    # controller_you_want.response = response
+    #
+    # controller_you_want.edit
+  end
```

Comment:
All of the above error out in some way either by failing with having the `html` field set so the controller calling `format.html` fails, or some variable is not set since a controller isnt instantiated (which is what I tried to do last but it didn't work)